app/models/location.rb

```diff
@@ -157,7 +157,7 @@ def status_at(date)
     return :incomplete unless confirmed
     return :deactivated if scheme.owning_organisation.status_at(date) == :deactivated ||
       open_deactivation&.deactivation_date.present? && date >= open_deactivation.deactivation_date
-    return :deactivating_soon if open_deactivation&.deactivation_date.present? && date < open_deactivation.deactivation_date
+    return :deactivating_soon if open_deactivation&.deactivation_date.present? && date < open_deactivation.deactivation_date && (startdate.blank? || date >= startdate)
```

Comment:
What if there is no startdate and there is a deactivation in the future tho? I think that should still be deactivating soon

We also have some scopes, if we're changing what deactivating_soon means here, we should probably be changing it in the scope as well (line 70)